src/components/Col/Col.js

```diff
@@ -10,13 +10,17 @@ const Col = styled(props =>
 )`
   // Initial component properties
   box-sizing: border-box;
-  flex: 0 0 auto;
-  display: block;
 
   ${props => themeProvider.breakpointsKeys(props).map(breakpoint => mediaQuery(props)[breakpoint]`
+    // Generated Display
+    ${CSSProperty(props, breakpoint, 'display')}
+
     // Generate gutter
     ${gutter.col(props, breakpoint)}
 
+    // Generate flex rule before width, this avoid override
+    ${CSSProperty(props, breakpoint, 'flex')}
```

Comment:
I think yes, if  we set for example different flex rule property like `flex: 1 2 auto;`. it's a plus.